### DISCUSSION TITLE
Have you checked what happens if you pick the event which has the dilepton Z mass closer to the Z boson value?

### DISCUSSION THREAD

**Discussion by ram1123** [ram1123/2l2Q_limitSettingTool#15] [Q&A] [answered]
@jialin-guo1 Any information on this?

**Comment by jialin-guo1** [comment, marked as answer]
before we moving to only keep those events whose ntight leptonts ==2, i did pick the di-leptons who has di-mass which is the closest to Z boson value

**Reply by ram1123** [reply]
Right. I did not think before adding the Filippo comment here. Since we are using Tight lepton == 2 so there is no question on how to choose the leptons.

Thanks
